src/structures/paging/page_table.rs

```diff
@@ -15,9 +15,6 @@ use bitflags::bitflags;
 pub enum FrameError {
     /// The entry does not have the `PRESENT` flag set, so it isn't currently mapped to a frame.
     FrameNotPresent,
-    /// The entry does have the `HUGE_PAGE` flag set. The `frame` method has a standard 4KiB frame
-    /// as return type, so a huge frame can't be returned.
-    HugeFrame,
```

Comment:
Removing this variant is a breaking change. Breaking changes should target the `next` branch. Alternatively, we could keep the variant, mark it as deprecated and add a notice explaining why we're no longer returning this variant.